Tests sporadically failing owing to __org_kroxylicious_testing_consistencyTest

**Describe the bug**

On main (d34746ca8967894afe1e6c48b846f995e5d794c8)  when running the whole test suite, I'm sporadically seeing this problem.

```
[WARNING] Tests run: 4, Failures: 0, Errors: 0, Skipped: 2, Time elapsed: 18.20 s -- in io.kroxylicious.proxy.ResilienceIT
[INFO]
[INFO] Results:
[INFO]
[ERROR] Failures:
[ERROR]   KrpcFilterIT.requestFiltersCanRespondWithoutProxying:163
Expecting empty but was: ["__org_kroxylicious_testing_consistencyTest"]
[INFO]
[ERROR] Tests run: 49, Failures: 1, Errors: 0, Skipped: 2
```



**To Reproduce**
Steps to reproduce the behavior:
1. Configure kroxylicious like this ...
2. Run command '....'
3. See error

**Expected behavior**
A clear and concise description of what you expected to happen.

**YAML files and logs**

Attach or copy paste your kroxylicious configuration YAML (with sensitive details
like broker address redacted).
Attach or copy and paste relevant logs.

**Additional context**
Add any other context about the problem here.

--- a/integrationtests/src/test/java/io/kroxylicious/proxy/KrpcFilterIT.java
+++ b/integrationtests/src/test/java/io/kroxylicious/proxy/KrpcFilterIT.java
@@ -159,7 +159,7 @@ public class KrpcFilterIT {
 
             // check no topic created on the cluster
             Set<String> names = admin.listTopics().names().get(10, TimeUnit.SECONDS);
-            assertThat(names).isEmpty();
+            assertThat(names).doesNotContain(TOPIC_1);
         }
     }
 
@@ -181,7 +181,7 @@ public class KrpcFilterIT {
 
             // check no topic created on the cluster
             Set<String> names = admin.listTopics().names().get(10, TimeUnit.SECONDS);
-            assertThat(names).isEmpty();
+            assertThat(names).doesNotContain(TOPIC_1);
         }
     }
 

